tensorflow/python/kernel_tests/relu_op_test.py

```diff
@@ -302,6 +302,97 @@ def testGradGradFloat64(self):
     self.assertLess(err, 1e-6)
 
 
+class SeluTest(test.TestCase):
+
+  def _npSelu(self, np_features):
+    scale = 1.0507009873554804934193349852946
+    scale_alpha = 1.7580993408473768599402175208123
+    return np.where(np_features < 0, scale_alpha * (np.exp(np_features) - 1),
+                    scale * np_features)
+
+  def testNpSelu(self):
```

Comment:
These numbers look off to me. The numerical values not the same as I get in my implementation.